src/SUnit-Core-Traits/TAssertable.trait.st

```diff
@@ -194,6 +194,15 @@ TAssertable >> denyCollection: actual equals: expected [
 		description: [ self unexpectedEqualityStringBetween: actual and: expected ]
 ]
 
+{ #category : #asserting }
+TAssertable >> denyCollection: actual hasSameElements: expected [
+	"Deny that a collection contains the same elements as the given collection. Order is not checked, only the presence/absence of elements."
+
+	self
+		deny: ((actual difference: expected) isEmpty or: [ (expected difference: actual) isEmpty ])
+		description: (String streamContents: [ :stream | stream nextPutAll: 'Given collections match!' ])
```

Comment:
Why `String streamContents:` and not directly: `'Given collections match!'` ?

Reply:
I think the implementation is not right or I'm misinterpreting the intention.
If you do: `self denyCollection: #(1 2) hasSameElements: #(1 2 4)` the test fails